page-faq.php

```diff
@@ -278,25 +310,31 @@
 </ul>
 
 <a name="closedapps"></a>
-<h3>Can I write closed source or proprietary apps for ownCloud?</h3>
+<h3>Can I write closed source or proprietary apps for ownCloud Server?</h3>
```

Comment:
Now that you also explain how to write external apps it doesn't make sense to add the "Server" part here.